src/process/dust/ccpr_dust_common_mod.F90

```diff
@@ -44,26 +44,26 @@ module CCPr_Dust_Common_Mod
    !!!>
    TYPE :: DustStateType
       ! Generic Variables for Every Process
-      Logical                         :: Activate            ! Activate Process (True/False)
-      INTEGER                         :: nDustSpecies        ! Number of dust processes
-      INTEGER                         :: SchemeOpt           ! Scheme Option
-      INTEGER, ALLOCATABLE            :: DustSpeciesIndex(:) ! Index of dust species
-      INTEGER, ALLOCATABLE            :: SpcIDs(:)           ! CATChem species IDs
+      Logical                         :: Activate                !< Activate Process (True/False)
+      INTEGER                         :: nDustSpecies            !< Number of dust processes
+      INTEGER                         :: SchemeOpt               !< Scheme Option
+      INTEGER, ALLOCATABLE            :: DustSpeciesIndex(:)     !< Index of dust species
+      INTEGER, ALLOCATABLE            :: SpcIDs(:)               !< CATChem species IDs
 
       ! Process Specific Parameters
-      REAL(fp), ALLOCATABLE           :: LowerBinRadius(:)         ! Lower bin radius        [m]
-      REAL(fp), ALLOCATABLE           :: UpperBinRadius(:)         ! Upper bin radius        [m]
-      REAL(fp), ALLOCATABLE           :: EffectiveRadius(:)        ! Effective radius        [m]
-      REAL(fp), ALLOCATABLE           :: DustDensity(:)            ! Dust density            [kg/m^3]
-      REAL(fp)                        :: BetaScaleFactor           ! Beta Scaling Parameter  [1]
-      REAL(fp)                        :: AlphaScaleFactor          ! Alpha Scaling Parameter [1]
-      REAL(fp), ALLOCATABLE           :: TotalEmission             ! Total emission          [kg/m^2/s]
-      REAL(fp), ALLOCATABLE           :: EmissionPerSpecies(:)     ! Emission per species    [kg/m^2/s]
+      REAL(fp), ALLOCATABLE           :: LowerBinRadius(:)       !< Lower bin radius        [m]
+      REAL(fp), ALLOCATABLE           :: UpperBinRadius(:)       !< Upper bin radius        [m]
+      REAL(fp), ALLOCATABLE           :: EffectiveRadius(:)      !< Effective radius        [m]
+      REAL(fp), ALLOCATABLE           :: DustDensity(:)          !< Dust density            [kg/m^3]
+      REAL(fp)                        :: GammaScaleFactor        !< Gamma Scaling Parameter  [1]
```

Comment:
@bbakernoaa seems like this was the issue, Beta was changed to Gamma in the structure here but not somewhere else